src/base/pages/Landing.vue

```diff
@@ -362,7 +362,7 @@ export default {
       return images
     },
     requireImage ({ fileName, ext, width, dirName } = {}) {
-      return require(`@/base/pages/images/${dirName}/${fileName}-${width}${ext}`)
+      return require(`@/base/pages/images/${dirName}/${fileName}-${width}${ext}?disableinline`)
```

Comment:
Maybe could use a comment here explaining why to disable inlining? And perhaps a link to the issue for a longer explanation/story.

Reply:
done